amenity-points.mss

```diff
@@ -1488,12 +1488,46 @@
     marker-fill: @airtransport;
   }
 
-  [feature = 'aeroway_aerodrome']['access' != 'private']['icao' != null]['iata' != null][zoom >= 10][zoom < 14],
-  [feature = 'aeroway_aerodrome'][zoom >= 11][zoom < 14] {
-    marker-file: url('symbols/aerodrome.12.svg');
-    marker-placement: interior;
-    marker-clip: false;
-    marker-fill: @airtransport;
+  [feature = 'aeroway_aerodrome']['access' = 'private'],
+  [feature = 'aeroway_aerodrome']['icao' = null],
+  [feature = 'aeroway_aerodrome']['iata' = null] {
+    [zoom >= 12][zoom < 17][way_pixels = null] {
+      marker-file: url('symbols/aerodrome.12.svg');
+      marker-placement: interior;
+      marker-clip: false;
+      marker-fill: @airtransport;
+      [zoom >= 15] { marker-file: url('symbols/aerodrome.15.svg'); }
+      [zoom >= 16] { marker-file: url('symbols/aerodrome.18.svg'); }
+    }
+    [zoom >= 12][zoom < 18][way_pixels <= 768000] {
```

Comment:
@imagico - are you ok with using way_pixels to stop rendering the icon/text sooner for large aerodromes which are mapped as areas, as in this line and in lines 1522, 3175 and 3226?